src/applications/gqrx/mainwindow.cpp

```diff
@@ -161,6 +161,7 @@ MainWindow::MainWindow(const QString& cfgfile, bool edit_conf, QWidget *parent)
     uiDockFft->toggleViewAction()->setShortcut(QKeySequence(Qt::CTRL | Qt::Key_F));
     uiDockAudio->toggleViewAction()->setShortcut(QKeySequence(Qt::CTRL | Qt::Key_A));
     uiDockBookmarks->toggleViewAction()->setShortcut(QKeySequence(Qt::CTRL | Qt::Key_B));
+    iq_tool->toggleViewAction()->setShortcut(QKeySequence(Qt::CTRL | Qt::Key_I));
```

Comment:
This is not required as the shortcut is already defined in the https://github.com/gqrx-sdr/gqrx/blob/79645d962a9c240bdd1e4f78b5780db86b2d069c/src/applications/gqrx/mainwindow.ui#L685
Assigning a shortcut for the second time result in non-working shortcut and Qt error "QAction::eventFilter: Ambiguous shortcut overload: Ctrl+I".